control/trajectory_follower_nodes/design/trajectory_follower-design.md

```diff
@@ -8,12 +8,12 @@ Generate control commands to follow a given Trajectory.
 
 This functionality is decomposed into three nodes.
 
-- @subpage lateral-controller-design : generates lateral control messages.
-- @subpage longitudinal-controller-design : generates longitudinal control messages.
-- @subpage latlon-muxer-design : combines the lateral and longitudinal control commands
+- [lateral-controller-design](lateral_controller-design.md) : generates lateral control messages.
+- [longitudinal-controller-design](longitudinal_controller-design.md) : generates longitudinal control messages.
+- [latlon-muxer-design](latlon_muxer-design.md) : combines the lateral and longitudinal control commands
   into a single control command.
 
-Core functionalities are implemented in the @subpage trajectory_follower-package-design package.
+Core functionalities are implemented in the [trajectory_follower](https://github.com/autowarefoundation/autoware.universe/blob/main/control/trajectory_follower/design/trajectory_follower-design.md) package.
```

Comment:
If it's in the same repository, it's better to use relative links.

Related: https://github.com/autowarefoundation/autoware-documentation/pull/116#discussion_r876735966

Reply:
I'm struggling with relative path...... 
All relative path are dead even if the docs is alive.
Could you give me some advise?

### Case 1:  [relative-path]/file-name
https://github.com/autowarefoundation/autoware.universe/commit/7c477738845b78fc317009295968e06faf44127f
[ [✖] ../../../trajectory_follower/design/trajectory_follower-mpc-design → Status: 400 ](https://github.com/autowarefoundation/autoware.universe/runs/6567020803?check_suite_focus=true#step:3:958)

when clicking trajectory_follower-mpc-design,

![Screenshot from 2022-05-24 13-46-47](https://user-images.githubusercontent.com/42209144/170001746-cbf82fda-2418-49a9-aad5-f98653fb7149.png)

#### **docs is alive.**
![Screenshot from 2022-05-24 13-47-07](https://user-images.githubusercontent.com/42209144/170001760-364654d1-f894-4ef0-9f50-f431ac4ef19f.png)


### Case 2:  [relative-path]/file-name.md
https://github.com/autowarefoundation/autoware.universe/commit/7c477738845b78fc317009295968e06faf44127f
[  [✖] ../../../trajectory_follower/design/trajectory_follower-design.md → Status: 400](https://github.com/autowarefoundation/autoware.universe/runs/6567020803?check_suite_focus=true#step:3:306)

when clicking trajectory_follow-design,

![Screenshot from 2022-05-24 13-47-21](https://user-images.githubusercontent.com/42209144/170001773-e6d3a0ff-d699-41a9-b512-35c0d80caab8.png)

docs is dead.
![Screenshot from 2022-05-24 13-47-25](https://user-images.githubusercontent.com/42209144/170001790-f5cac39d-8d30-47c3-a0eb-05a72c20fc27.png)


### Case 3:  [relative-path]/file-name.md/#heading
https://github.com/autowarefoundation/autoware.universe/pull/950/commits/313c1d818afb74bc32e485d8716af4b93cddcc59
[[✖] ../../../trajectory_follower/design/trajectory_follower-design.md/#trajectory-follower](https://github.com/autowarefoundation/autoware.universe/runs/6570597117?check_suite_focus=true#step:3:300)

docs is dead. 

### Case 4: [relative-path]/file-name/file-name.md#heading
https://github.com/autowarefoundation/autoware.universe/pull/950/commits/313c1d818afb74bc32e485d8716af4b93cddcc59
[[✖] ../../../trajectory_follower/design/trajectory_follower-mpc-design/trajectory_follower-mpc-design.md#mpc-trajectory-follower](https://github.com/autowarefoundation/autoware.universe/runs/6570597117?check_suite_focus=true#step:3:951)

docs is dead.

### Case 5: [relative-path]/file-name#heading
https://github.com/autowarefoundation/autoware.universe/pull/950/commits/a520ccfe39b2bd7287340e19baca5aa88ed04c89
[  [✖] ../../../trajectory_follower/design/trajectory_follower-mpc-design#mpc-trajectory-follower → Status: 400](https://github.com/autowarefoundation/autoware.universe/runs/6571088686?check_suite_focus=true#step:3:958)

#### **docs is alive!**

### Case6: [relative-path]/file-name.md#heading
https://github.com/autowarefoundation/autoware.universe/pull/950/commits/a520ccfe39b2bd7287340e19baca5aa88ed04c89
[[✖] ../../../trajectory_follower/design/trajectory_follower-design.md#trajectory-follower → Status: 400](https://github.com/autowarefoundation/autoware.universe/runs/6571088686?check_suite_focus=true#step:3:306)

docs is dead.

Reply:
Okay, I'll try it!

Reply:
Fixed in d0385a7c7f33bb59836ec65d7290828c41b47276.

The points are:
- The depth was wrong.
- You need `.md`.
- It's good to check with your IDE first, not with MkDocs.

Reply:
Thank you!

> It's good to check with your IDE first,

I'll keep in mind.